dockerfiles/docker-compose-assets.yml

```diff
@@ -0,0 +1,8 @@
+version: '3'
```

Comment:
Why is this a different file? I find it nicer having everything in 1 place, so we don't have to check 6 different files for docker config.

Reply:
The nodejs container is not needed to run the development instance (and vice-versa). We use the same pattern here than the one for search/ext-them, which are optional as well.